package.json

```diff
@@ -32,55 +32,56 @@
   ],
   "homepage": "https://www.jhipster.tech",
   "peerDependencies": {
-    "axios": "*",
-    "dayjs": "^1.10.4",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-hook-form": "^7.30.0",
-    "reactstrap": "^9.0.3"
+    "axios": "0.27.2",
+    "dayjs": "1.11.3",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-hook-form": "7.30.0",
+    "reactstrap": "9.1.3"
   },
   "peerDependenciesMeta": {
     "reactstrap": {
       "optional": true
     }
   },
   "dependencies": {
-    "lodash": "^4.17.21",
+    "lodash": "4.17.21",
     "numeral": "^2.0.6",
     "sanitize-html": "^2.3.1",
-    "tslib": "^2.3.1"
+    "tslib": "2.4.0"
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.12.0",
-    "@testing-library/react": "^13.1.1",
+    "@testing-library/react": "13.3.0",
     "@types/jest": "^27.0.2",
-    "@types/react": "^18.0.5",
-    "@types/react-dom": "^18.0.1",
+    "@types/react": "18.0.14",
+    "@types/react-dom": "18.0.5",
     "@types/sanitize-html": "^2.3.1",
-    "@types/webpack-env": "^1.16.3",
-    "@typescript-eslint/eslint-plugin": "^5.20.0",
-    "@typescript-eslint/parser": "^5.20.0",
-    "axios": "^0.21.1",
-    "dayjs": "^1.10.4",
-    "eslint": "^8.13.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-react": "^7.22.0",
-    "husky": "^6.0.0",
-    "jest": "^27.0.3",
-    "jest-junit": "^12.1.0",
-    "lint-staged": "^11.0.0",
-    "prettier": "^2.2.1",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-hook-form": "^7.30.0",
-    "reactstrap": "^9.0.3",
-    "redux": "^4.0.5",
-    "rimraf": "^3.0.2",
-    "sinon": "^11.1.1",
-    "ts-jest": "^27.0.2",
-    "typescript": "^4.3.2"
+    "@types/webpack-env": "1.17.0",
```

Comment:
Fixed version at devDependencies is not really useful, package-lock exists for this.